libs/SoloPlay/Functions/MuleloggerOverrides.js

```diff
@@ -100,7 +100,7 @@ MuleLogger.logItem = function (unit, logIlvl, type = "Player") {
 	}
 
 	switch (unit.quality) {
-	case sdk.itemquality.Set:
+	case item.set:
```

Comment:
This case structure needs to be reverted back, it's checking the numerical value instead of the boolean one

Reply:
done